src/feature.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

Reply:
> * [ ]  You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

Hi. Thanks for the review. I would get just one specific eslint error pertaining to the import syntax, which is why I disabled it. I think that's reasonable.